.github/workflows/release.yml

```diff
@@ -111,6 +111,17 @@ jobs:
         with:
           name: oso_static_library
           path: polar-c-api/polar.h
+      - if: runner.os == 'Windows'
```

Comment:
At this point I feel like we are better off splitting these into separate jobs.

Reply:
Yeah I had the same thought, I'm gonna do that in a followup.